spec/board_spec.rb

```diff
@@ -13,125 +18,80 @@
+        @helper.fill_board(@board, [3,4,5], %w[x x x])
+        expect(@board.is_horizontal_win?).to be true
       end
 
       it "detects third winning combination" do
-        board = Board.new(3)
-        board.make_move(6, 'x')
-        board.make_move(7, 'x')
-        board.make_move(8, 'x')
-
-        expect(board.is_horizontal_win?).to be true
+        @helper.fill_board(@board, [6,7,8], %w[x x x])
+        expect(@board.is_horizontal_win?).to be true
       end
     end
     context "vertical wins" do
       it "detects first winning combination" do
-        board = Board.new(3)
-        board.make_move(0, 'x')
-        board.make_move(3, 'x')
-        board.make_move(6, 'x')
-
-        expect(board.is_vertical_win?).to be true
+        @helper.fill_board(@board, [0,3,6], %w[x x x])
+        expect(@board.is_vertical_win?).to be true
       end
 
       it "detects second winning combination" do
-        board = Board.new(3)
-        board.make_move(1, 'x')
-        board.make_move(4, 'x')
-        board.make_move(7, 'x')
-
-        expect(board.is_vertical_win?).to be true
+        @helper.fill_board(@board, [1,4,7], %w[x x x])
+        expect(@board.is_vertical_win?).to be true
       end
 
       it "detects third winning combination" do
-        board = Board.new(3)
-        board.make_move(2, 'x')
-        board.make_move(5, 'x')
-        board.make_move(8, 'x')
-
-        expect(board.is_vertical_win?).to be true
+        @helper.fill_board(@board, [2,5,8], %w[x x x])
+        expect(@board.is_vertical_win?).to be true
       end
     end
     context "diagonal wins" do
       it "detects first winning combination" do
-        board = Board.new(3)
-        board.make_move(0, 'x')
-        board.make_move(4, 'x')
-        board.make_move(8, 'x')
-
-        expect(board.is_diagonal_win?).to be true
+        @helper.fill_board(@board, [0,4,8], %w[x x x])
+        expect(@board.is_diagonal_win?).to be true
       end
 
       it "detects second winning combination" do
-        board = Board.new(3)
-        board.make_move(2, 'x')
-        board.make_move(4, 'x')
-        board.make_move(6, 'x')
-
-        expect(board.is_diagonal_win?).to be true
+        @helper.fill_board(@board, [2,4,6], %w[x x x])
+        expect(@board.is_diagonal_win?).to be true
       end
     end
   end
   context "detects win on the board" do
     it "board has winner" do
-      board = Board.new(3)
-      board.make_move(2, 'x')
-      board.make_move(4, 'x')
-      board.make_move(6, 'x')
-
-      expect(board.has_winner?).to be true
+      @helper.fill_board(@board, [2,4,6], %w[x x x])
+      expect(@board.has_winner?).to be true
     end
 
     it "board has no winner" do
-      board = Board.new(3)
-      board.make_move(2, 'x')
-      board.make_move(4, 'o')
-      board.make_move(6, 'x')
-
-      expect(board.has_winner?).to be false
+      @helper.fill_board(@board, [2,4,6], %w[x o x])
+      expect(@board.has_winner?).to be false
```

Comment:
RSpec has a funny matcher for boolean, which lets you rewrite this line as:

```rb
expect(@board).to be_has_winner
```

It will look for a method `has_winner?` on `@board`.

Reply:
interesting.. 👍